.circleci/config.yml

```
@@ -102,9 +103,13 @@ workflows:
       - release:
           filters:
             branches:
-              only: master
+              ignore: /.*/
+            tags:
+              only: /^v.*/
```

```suggestion
              only: /^v\.*/
```
Pretty sure you want to match a dot here.